From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: ZHANG Yuntian <yt@radxa.com>
Date: Wed, 25 Jun 2025 12:01:02 +0800
Subject: [PATCH] t23x: dts: update product model

Signed-off-by: ZHANG Yuntian <yt@radxa.com>
---
 nv-platform/tegra234-p3768-0000+p3767-0005-nv.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/nv-platform/tegra234-p3768-0000+p3767-0005-nv.dts b/nv-platform/tegra234-p3768-0000+p3767-0005-nv.dts
index 59560df..f1a1d6f 100644
--- a/nv-platform/tegra234-p3768-0000+p3767-0005-nv.dts
+++ b/nv-platform/tegra234-p3768-0000+p3767-0005-nv.dts
@@ -7,6 +7,8 @@
 #include "tegra234-p3768-0000+p3767-xxxx-nv-common.dtsi"
 
 / {
+	model = "Radxa Airbox Orin";
+
 	bus@0 {
 		host1x@13e00000 {
 			nvdla0@15880000 {
-- 
2.49.0

